inclusive-communication.md

```diff
@@ -0,0 +1,44 @@
+# Guidelines for inclusive communication
+
+The purpose of these guidelines are to provide some best practices and examples of inclusive communication. Just like many other best practices, you may be referred to these guidelines by your colleagues as part of code reviews, feedback sessions, document comments, and retrospectives, etc, so please do take time to digest them effectively. They are relevant to all forms of communication including verbal, written, drawing and code.
+
+Please do submit any edits you feel necessary, and these will be reviewed and merged by a member of the EM team.
+
+## Why is inclusive language important?
+
+*   Makes people feel naturally included
+*   Doesn’t trigger memories of someones’ prior trauma
+*   Brings more individuals within the perceived normality
+*   Helps overcome rather than cement peoples’ cognitive biases
+
+## Some examples
+
+#### From [Twitter Engineering on 2 July](https://twitter.com/TwitterEng/status/1278733305190342656)
+
+<img src="Eb74mBPU0AAct0s.jpeg" alt="https://twitter.com/TwitterEng/status/1278733305190342656"/>
```

Comment:
This isn't showing in the [rendered](https://github.com/guardian/our-engineering-culture/blob/12d36d5e521668b4c64e54c4316532b963ef00b6/inclusive-communication.md) markdown as the image 404s. Rather than copy the image across, could we use a table?

```suggestion
| **Non-inclusive** | **Alternatives** |
|---|---|
| Whitelist | Allowlist |
| Blacklist | Denylist |
| Master/slave | Leader/follower, primary/replica, primary/standby |
| Grandfathered | Legacy status |
| Gendered pronouns (e.g. guys) | Folks, people, you all, y'all |
| Gendered pronouns (e.g. he/him/his) | They, them, their |
| Man hours | Person hours, engineering hours |
| Sanity check | Quick check, confidence check, coherence check |
| Dummy value | Placeholder value, sample value |
```